guides/doc-Content_Management_Guide/topics/Managing_Content_Views.adoc

```diff
@@ -443,17 +443,17 @@ To resolve package dependencies by default, complete the following steps:
 . In the {Project} web UI, navigate to *Administer* > *Settings* and click the *Content* tab.
 . Locate the *Content View Dependency Solving Default*, and select *Yes*.
 
-You can also set the default level of dependency resolution that you want.
-You can select between adding packages to solve dependencies only if the required package does not exist, or to add the latest packages to resolve the dependency even if the package exists in the repository.
-
-To set the default level of dependency resolution, complete the following steps:
-
-. In the {Project} web UI, navigate to *Administer* > *Settings* and click the *Content* tab.
-. Locate the *Content View Dependency Solving Algorithm* and select one of the following options:
+.Considerations
+. The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View, so `{package-update-project}` is not guaranteed to work.
+. Packages in a Content View can have conflicting dependencies, which is expected.
+* A -> B(1.1)
+* C -> B(1.2)
+Where A, B and C are example packages where package A has a dependency on version 1.1 of package B, and Package C has a dependency on Version 1.2 of package B.
```

Comment:
I would drop or move the two lines above since the user doesn't know what this is supposed to mean yet.

> Imagine a scenario where A, B and C are example packages where package A has a dependency on version 1.1 of package B, and Package C has a dependency on Version 1.2 of package B.

Reply:
If there are no more comments on this. I'll ask for it to be merged.